manifests/foreman_proxy_content.pp

```diff
@@ -30,7 +33,14 @@
   class { '::certs::qpid_router':   hostname => $foreman_proxy_fqdn, cname => $foreman_proxy_cname }
   class { '::certs::qpid_client':   hostname => $foreman_proxy_fqdn, cname => $foreman_proxy_cname }
 
-  certs::tar_create { $certs_tar:
-    subscribe => Class['certs::puppet', 'certs::foreman', 'certs::foreman_proxy', 'certs::qpid', 'certs::qpid_router', 'certs::apache', 'certs::qpid_client'],
+  if !$generate_repomd_gpg {
+    certs::tar_create { $certs_tar:
+      subscribe => Class['certs::puppet', 'certs::foreman', 'certs::foreman_proxy', 'certs::qpid', 'certs::qpid_router', 'certs::apache', 'certs::qpid_client'],
+    }
+  } else {
+    class { '::certs::repomd_gpg': hostname => $foreman_proxy_fqdn }
+    certs::tar_create { $certs_tar:
+      subscribe => Class['certs::puppet', 'certs::foreman', 'certs::foreman_proxy', 'certs::qpid', 'certs::qpid_router', 'certs::apache', 'certs::qpid_client', 'certs::repomd_gpg'],
```

Comment:
You can build the `subscribe` list dynamically as I have in another PR https://github.com/theforeman/puppet-certs/pull/199/files#diff-1aca67488955df8f44fd9c808ade15f1R31

Reply:
Ok, I'll change this.